### ISSUE TITLE
better starter example docs

### ISSUE THREAD

**Issue by raould** [jsverify/jsverify#149]
The current examples in the README leave me more confused than directed. Is there anywhere a really simple set of docs that start with something really simple and basic and then work their way up? (The very first example is, I think, kind of ridiculously meta.)

**Comment by raould** [comment]
Things like https://github.com/mostalive/property-based-tutorial/blob/master/exercises/js/js-intro.md are I posit better intros. At least link to some of those kinds of things up front pretty please?

**Comment by einpaule** [comment]
I agree with @raould that a gentler introduction would be very helpful to people trying to get into property based testing. I think a pointer to that tutorial page would already be helpful, the readme doesn't necessarily take over any of the contents.

**Comment by octagonal** [comment]
A less steep intro would be much appreciated by me as well
